### PR TITLE
Improves documentation.

### PR DESCRIPTION
**Pull request by francocipollone**
# 🎉 New feature

Related to https://github.com/maliput/maliput_infrastructure/issues/292

## Summary
- Adds a proper readme.

## Checklist
- [x] Signed all commits for DCO
- [ ] Added tests
- [ ] Added example and/or tutorial
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] Consider updating Python bindings (if it affects the public API)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,97 @@
...
+
+[Getting Started](https://maliput.readthedocs.io/en/latest/getting_started.html) page is a good place for starting to see Maliput's capabilities and how to use a Maliput backend for getting a road network.
+
+ - [maliput_integration](https://github.com/maliput/maliput_integration): Concentrates applications created for maliput. See [maliput_integration's tutorials](https://maliput.readthedocs.io/en/latest/html/deps/maliput_integration/html/integration_tutorials.html). These applications allow to select `maliput_osm` as the backend.
+
+## Installation
+
+### Supported platforms
+
+Ubuntu Focal Fossa 20.04 LTS.
```
Comment (agalbachicar) [inline README.md:37]:
```suggestion
Ubuntu Focal Fossa 20.04 LTS, ROS2 Foxy.
```

File: README.md [hunk 2]
```diff
@@ -1 +1,97 @@
...
+    cd colcon_ws/src
+    git clone https://github.com/maliput/maliput_osm.git
+    ```
+
+3. Install package dependencies via `rosdep`
+    ```
+    export ROS_DISTRO=foxy
+    ```
+    ```sh
+    rosdep update
```
Comment (agalbachicar) [inline README.md:69]: I've just realized that there is no step to bring in dependencies. Shall we add a repos file in this guide?

**Review by agalbachicar — CHANGES_REQUESTED** [review]
PTAL

File: README.md (hunk 2 above)
Comment (francocipollone) [reply README.md:69]: Not necesary. These instructions are there to show how you can build this package from source but you are still depending on rosdep for installing the dependencies.

Let's say that you want to contribute to maliput_osm then you simply clone `maliput_osm`, then you rosdep install the dependencies (maliput and maliput_sparse) and you can work on maliput_osm without the necessity to install everything from source, just what you need.

**Review by agalbachicar — APPROVED** [review]
LGTM

**Pull request merged**